### PR TITLE
Add Percentage column with list

### LINKED ISSUE #126: Show percentage
While using the app, I found that sometimes I need to finish pomodoro timer early. In that case, I want to record how much did I spend - 70%, 50% for example. Let's make this option

This is adding new column named percentage

### PR DESCRIPTION
**Pull request by TheRustyPickle**
**Updates:**

- Fixed `create` and `queue` command not handling `-w` and`-b` and instead was using default values for all cases due to `contains_id` method returning `true`.
- Added `percentage` column when `-p` flag is passed with `ls`

Note: When using UDS, the app does not show the percentage column yet. Resolves #126.

Mistakenly closed #162 while resolving a conflict.

### CODE REVIEW

**Comment by 24seconds** [comment]
Hello @TheRustyPickle , could you elaborate this with some example? I don't understand.
> Fixed create and queue command not handling -w and-b and instead was using default values for all cases due to contains_id method returning true.

**Comment by TheRustyPickle** [comment]
Hello @24seconds, looks like in a recent pr, `default_value` was removed from from the args `work` and `break`. Due to the default values, `contains_id` always returned `true` on [this line ](https://github.com/24seconds/rust-cli-pomodoro/blob/main/src/command/util.rs#L10) in all cases. That part is no longer valid as it was removed.

File: src/notification/mod.rs [hunk 1]
```diff
@@ -653,4 +677,91 @@ mod tests {
 
         assert!(notification.is_err());
     }
+
+    #[test]
+    fn test_work_percentage() {
```
Comment (24seconds) [inline src/notification/mod.rs:682]: super!

**Review by 24seconds — APPROVED** [review]
LGTM!

File: src/command/application.rs [hunk 2]
```diff
@@ -232,6 +238,6 @@ mod tests {
         let matches =
             add_args_for_create_subcommand(cmd).get_matches_from("myapp -d".split_whitespace());
 
-        assert!(matches.contains_id("default"));
+        assert!(matches.get_flag("default"));
```
[the remark this answers is not in the document]
Comment (24seconds) [reply src/command/application.rs:241]: Could you take a look this? 🙇

[the remark this answers is not in the document]
Comment (TheRustyPickle) [reply src/command/application.rs:241]: Sure! How about a test for `break` too, similar to the work test?

[the remark this answers is not in the document]
Comment (24seconds) [reply src/command/application.rs:241]: good!

**Pull request merged**